consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/exception/CMCException.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.moj.sdt.cmc.consumers.exception;
+
+public class CMCException extends RuntimeException {
+
+    public CMCException(String message, Throwable cause) {
```

Comment:
Might want to add the serialVersionUID field to this class.  It was flagged as a problem by the PMD plugin when I created an exception class that inherited from RuntimeException (see https://pmd.github.io/latest/pmd_rules_java_errorprone.html#missingserialversionuid).  We don't seem to be using that plugin in civil-sdt though.

If you want to add it you can get IntelliJ to do it by temporarily making the class implement the Serializable interface, using the context action to add it, then removing the implements bit.

Reply:
done

Reply:
The serialVersionUID field hasn't been added to this class, but it has been added to XmlToObjectConverter.  Might've been added to XmlToObjectConverter by mistake.

Reply:
removed from XmlToObjectConverter and added to CMCException